client/my-sites/current-site/index.jsx

```diff
@@ -39,13 +39,12 @@ class CurrentSite extends Component {
 		anySiteSelected: PropTypes.array,
 		forceAllSitesView: PropTypes.bool,
 		sidebarIsCollapsed: PropTypes.bool,
-		expandSidebar: PropTypes.func.isRequired,
 		isNavUnificationEnabled: PropTypes.bool.isRequired,
 	};
 
 	switchSites = ( event ) => {
 		if ( this.props.isNavUnificationEnabled && this.props.sidebarIsCollapsed ) {
```

Comment:
Since this code is also used in the `render()` method I wonder if it will be more DRY and readable to move it to it's own method:

```js
unifiedNavSidebarIsCollapsed = () => {
	return this.props.isNavUnificationEnabled && this.props.sidebarIsCollapsed;
}
```

Reply:
Excellent point! Done e524308